File: answers/go/write.go
package write_aoc_answers

import (
	"fmt"
	"os"
)

func WriteAnswers(first, second string) error {
	output := formatAnswers(first, second)
	err := os.WriteFile("answers.txt", []byte(output), 0666)
	if err != nil {
		return err
	}

	return nil
}

func formatAnswers(first, second string) string {
	return fmt.Sprintf("Answer 1:\n%s\n\nAnswer 2:\n%s\n", first, second)
}

File: answers/c++/write.cpp
#include <iostream>
#include <fstream>
#include <string>
#include "write.h"

using namespace std;

void WriteAnswers(string answer1, string answer2)
{
    ofstream answersTxt;
    answersTxt.open("answers.txt");
    answersTxt << "Answer 1:" << endl
               << answer1 << endl
               << endl
               << "Answer 2:" << endl
               << answer2 << endl;
    answersTxt.close();
}
